mycore-mods/src/main/resources/xsl/solr/indexing/mods-dynamicfields-3.xsl

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="3.0" 
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  xmlns:mods="http://www.loc.gov/mods/v3" 
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"
+  xmlns:mcrmods="http://www.mycore.de/xslt/mods">
+  
+  <xsl:import href="xslImport:solr-document-3:solr/indexing/mods-mycoreobject-dynamicfields-3.xsl" />
```

Comment:
This needs to be `solr/indexing/mods-dynamicfields-3.xsl`

Reply:
you are right - fixed!